google_compute_engine_oslogin/authorized_keys/authorized_keys.cc

```diff
@@ -36,14 +36,17 @@ int main(int argc, char* argv[]) {
   }
   std::stringstream url;
   url << kMetadataServerUrl << "users?username=" << UrlEncode(argv[1]);
-  string user_response = HttpGet(url.str());
+  string user_response;
+  if (!HttpGet(url.str(), &user_response)) {
+    return 1;
+  }
   if (user_response.empty()) {
     // Return 0 if the user is not an oslogin user. If we returned a failure
```

Comment:
So it actually returns 404, and the response is non-empty, with an error message. This is different from behavior I had tested with a long time ago. It may make sense to return the status code, and have guest code respond to that appropriately.